extension/script/docs-rs.js

```diff
@@ -105,12 +105,32 @@ document.addEventListener("DOMContentLoaded", async () => {
     }
 });
 
-async function enhanceFeatureFlagsMenu(menu) {
+async function getFeatureFlagsMenuData() {
     // Use rawCrateName to fetch the Cargo.toml, otherwise will get 404.
     let crateAPIURL = `https://crates.io/api/v1/crates/${rawCrateName}/${crateVersion}`;
     let response = await fetch(crateAPIURL);
     let content = await response.json();
     let features = parseCargoFeatures(content);
+    window.sessionStorage.setItem('features', JSON.stringify(features));
```

Comment:
The key should be `{crateName}-{crateVersion}`. 😅 Save the `features` and `optionalDependencies` into the same key.